script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -571,109 +573,107 @@ I have no mental capacity left to allocate to any other eventualities.
 [sha:0df06b142239c91ae758079373e614b3eefa6017]{
 -- Page 693, Offset 10528.
 -- だからもう、降り始めた俺には<対|も><応|の>を考える機能はない。
-So now that I've started to descend,
- I don't have the ability to think about how to respond to him.
+Now that I've initiated the descent,
+ I've lost the ability to formulate thoughts about how to react to him.
 }
 [sha:c057faa764c9a44cf6a342af0afa3732afa36dc0]{
 -- Page 694, Offset 10529.
 -- もしマイナスが二ヵ所に留まらない場合は？
-// "in that period" added to clarify that the problem is his clock-rate and the
-// limited time period
-What if I have to handle more than just two of those do-or-die moments in that period?
+What if there are more than two of these critical junctures?
 }
 [sha:a246cbce56656d812d3d22aa8c69c334bdbeef13]{
 -- Page 694, Offset 10530.
 -- それは考慮しない。三ヵ所もの『死』を超える余力は俺にはない。その時はヤツの勝ちだ。ヤツの異常性が、俺の想像を上回っているだけの話。
-I do not consider that.
- I don't have the capacity to overcome three 'deaths' in my mind.
+I won't consider it.
+ I don't have it in me to overcome three of them.
  If that's how it plays out, victory is his.
- It would just mean that his power is stronger than I can imagine.
+ That would just mean his powers were stronger than I imagined.
 }
 [sha:75680ba2b152fc7b3c04412eff6c4bb7e01ba93f]{
 -- Page 695, Offset 10531.
 -- 氷の槍は折り重なるように、落下する獲物に殺到する。
-A dense flurry of ice spears shoots toward the falling prey.
+A dense flurry of ice spears bombard the falling prey.
 }
 [sha:ba9c2a1086ba55450f0e1c97fe70b959fa0c1def]{
 -- Page 695, Offset 10532.
 -- まるで獣の<顎|あぎと>だ。恐ろしいがどうでもいい。
-It's like the jaws of a beast. It's terrifying, but it doesn't matter.
+Not unlike the jaws of a beast. Terrifying, but inconsequential.
 }
 [sha:9f532dbc35cebc5dc45cc94b45e9300a64065ff8]{
 -- Page 695, Offset 10533.
 -- そんな、ただ本能に因っただけの<咀嚼|そしゃく>など知った事か。
-I wonder if it's thanks to his instincts that he anticipated this.
+I don't give a damn about such an instinctual display of voraciousness.
 }
 [sha:0032cdf383b9b0e7c3f16d1068f5192c946dc6f9]{
 -- Page 695, Offset 10534.
 -- 俺がそうくると確信し、そうなれば殺されると恐れた迎撃は、
 Convinced that I would come for him,
- and terrified that I would kill him when I did...
+ and terrified that I would kill him when I did, he counterattacks―――
```

Comment:
I just realized this is probably wrong when working on the above line. I had my doubts about it when I was editing the first time, but kept it as is.
 It leads to 一つめの<致|マ><死|イ><地|ナ><点|ス>。so in full:

>俺がそうくると確信し、そうなれば殺されると恐れた迎撃は、一つめの<致|マ><死|イ><地|ナ><点|ス>。

Which is why the ―――10メートル。 is sort of inserted in there with the ――― to indicate an it's not part of the sentence.

Which def makes this more like:
>I was convinced it would come, which is why the counterattack I feared would kill me...
>Is the coming of the first critical juncture.

But the way the page goes over complicates things, because these two sentences need to be connected, but at the same time able to stand on their own. Or at least the second one does.

At the same time this also connect to the previous sentence, sort of contrast wise.
To give an idea:
"I don't care about these shitty spears because -> The attack I was actually fearing is here > 10 meters, first juncture.

Reply:
Makes sense. Sth like?
> Not unlike the jaws of a beast. Terrifying, but inconsequential.
> His feral snapping at my heels barely merits a second thought.
> Not when I know what's yet to come――the counterattack I feared would kill me.
> Ten meters.
> The first critical juncture.